core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -116,9 +120,12 @@ private[spark] class ExecutorAllocationManager(
   // TODO: The default value of 1 for spark.executor.cores works right now because dynamic
   // allocation is only supported for YARN and the default number of cores per executor in YARN is
   // 1, but it might need to be attained differently for different cluster managers
-  private val tasksPerExecutor =
+  private val tasksPerExecutorForFullParallelism =
     conf.getInt("spark.executor.cores", 1) / conf.getInt("spark.task.cpus", 1)
 
+  private val fullExecutorAllocationDivisor =
+    conf.getDouble("spark.dynamicAllocation.fullExecutorAllocationDivisor", 1.0)
```

Comment:
forgot about this earlier but this should really be a config similar to  DYN_ALLOCATION_MIN_EXECUTORS